app/models/submission.rb

```diff
@@ -36,4 +36,8 @@ def all_entries_valid?
   def sheet_names
     entries.distinct.pluck(Arel.sql("source->>'sheet'"))
   end
+
+  def report_no_business?
```

Comment:
oops, didn't realise this method was new in this PR. It needs some tests. Here are some, to be added to submission_spec.rb:36

```ruby
  describe '#report_no_business?' do
    it 'returns false if submission has files' do
      expect(FactoryBot.create(:submission_with_validated_entries).report_no_business?).to be(false)
    end
    it 'returns true if submission has no files' do
      expect(FactoryBot.create(:submission).report_no_business?).to be(true)
    end
  end
```